src/system/os.py

```diff
@@ -23,3 +23,14 @@ def current_platform() -> str:
         return "windows"
     else:
         return subprocess.check_output(["uname", "-s"]).decode().strip().lower()
+
+
+def rpm_architecture_alt(architecture: str) -> str:
```

Comment:
I would just call this `rpm_architecture`, lose the `alt`.

Reply:
fixed.